backend/src/StarResults.cjs

```diff
@@ -503,8 +509,19 @@ export function splitPR(candidates, scores, nWinners) {
       split_point
     );
   }
+  //Moving weighted sum totals to matrix for easier plotting
+  var weightedSumsData = []
+  for (let c = 0; c < num_candidates; c++) {
+    weightedSumsData.push(Array(weightedSumsByRound.length).fill(0))
+  }
+  weightedSumsByRound.map((weightedSums,i) => {
+    weightedSums.map((weightedSum,j) => {
+      let candidate = candidatesByRound[i][j]
+      weightedSumsData[candidate.index][i] = weightedSum*5
+    })
+  })
   losers = candidates;
-  return { winners, losers, others, ties, debuginfo };
+  return { winners, losers, others, ties, debuginfo,weightedSumsData };
```

Comment:
I'm not familiar with this syntax, the braces look like a dictionary, but it's not using the key:value format. I tried looking it up, but I probably wasn't searching for the right thing